looprpc/client.proto

```diff
@@ -37,9 +37,31 @@ service SwapClient {
 
     /** loop: `monitor`
     Monitor will return a stream of swap updates for currently active swaps.
-    TODO: add MonitorSync version for REST clients.
     */
-    rpc Monitor (MonitorRequest) returns (stream SwapStatus);
+    rpc Monitor (MonitorRequest) returns (stream SwapStatus) {
+        option (google.api.http) = {
+            get: "/v1/loop/monitor"
+        };
+    }
+
+    /**
+    ListSwaps returns a list of all currently known swaps and their current
+    status.
+    */
+    rpc ListSwaps (ListSwapsRequest) returns (ListSwapsResponse) {
+        option (google.api.http) = {
+            get: "/v1/loop/swaps"
+        };
+    }
+
+    /**
+    SwapInfo returns all known details about a single swap.
+    */
+    rpc SwapInfo (SwapInfoRequest) returns (SwapStatus) {
```

Comment:
So we specifically want an endpoint that is going to be polled and not extend `monitor` to just return updates for a single swap?

Reply:
Yes, because the monitor endpoint is streaming which is difficult to handle in REST. And it also doesn't return all older/completed swaps.